exercises/high-scores/high-scores.spec.js

```diff
@@ -13,51 +13,31 @@ describe('High Scores Test Suite', () => {
 
   xtest('Highest score', () => {
```

Comment:
can you please change this description to `Personal best` to match the canonical data?

Reply:
@tejasbubane done.